src/features/users/users-service.ts

```diff
@@ -0,0 +1,22 @@
+import { AxiosResponse } from "../../app/models/AxiosResponse";
+import { User } from "../../app/models/User";
+import { axiosDelete } from "../../axiosHelpers/axiosDeleteHelper";
+import { axiosPost } from "../../axiosHelpers/axiosPostHelper";
+import { axiosPut } from "../../axiosHelpers/axiosPutHelper";
+
+
+export class UserService {
+  addUser = async(postBody: User): Promise<AxiosResponse> => {
```

Comment:
Adjust all the AxiosResponse tags to pass the type using the new generic setup.

`addUser = async(postBody: User): Promise<AxiosResponse**<User>**> => {
    return await axiosPost(`users`, postBody)
  }`